CMakeLists.txt

```diff
@@ -590,6 +590,10 @@ if(BUILD_ONNX_PYTHON)
     target_link_libraries(onnx_cpp2py_export
                           PRIVATE "-Wl,--whole-archive" $<TARGET_FILE:onnx>
                                   "-Wl,--no-whole-archive")
+    # Prevent "undefined symbol: _ZNSt10filesystem7__cxx114path14_M_split_cmptsEv" on gcc 8
```

Comment:
undefined symbol is ``std::filesystem::__cxx11::path::_M_split_cmpts()`` which is more clear that the name in the comment.  ``python -c "import cxxfilt;print(cxxfilt.demangle('<symbol>')"``.

Reply:
The name in the comment was what showed in the terminal output when the build failed. I figured if someone hit this, they would then google or search our repo for that form.

Would you prefer I add the demanged version to the comment? So something like
```
# Prevent "undefined symbol: _ZNSt10filesystem7__cxx114path14_M_split_cmptsEv" 
# (std::filesystem::__cxx11::path::_M_split_cmpts()) on gcc 8
```
Or I could drop the mangled version from the comment entirely?